packages/flutter_test/lib/src/animation_sheet.dart

```diff
@@ -0,0 +1,341 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math' as math;
+import 'dart:ui' as ui;
+
+import 'package:flutter/rendering.dart';
+import 'package:flutter/scheduler.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+/// Records the frames of an animating widget, and later displays it in an
+/// animation sheet.
+///
+/// This class does not support Web, because taking screenshots is unsupported,
+/// which is necessary to build an animation sheet. Tests thatuse this class must
+/// be noted with `skip: isBrowser`.
+/// (https://github.com/flutter/flutter/issues/56001)
+///
+/// Using this class takes the following steps:
+///
+///  * Create an instance of this class.
+///  * Pump frames that render the target widget wrapped in [record]. Every frame
+///    that has `recording` being true will be recorded.
+///  * Optionally, adjust the size of the test view port to the
```

Comment:
```suggestion
///  * Optionally, adjust the size of the test viewport to the
```
What is the size of the test viewport? Can we link to some further information about that here?